# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/neofetch/hotfix-hppa.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- neofetch-7.1.0/neofetch.vanilla	2025-10-04 18:35:55.947950738 +0200
+++ neofetch-7.1.0/neofetch	2025-10-04 18:40:21.570000000 +0200
@@ -2073,7 +2073,7 @@
             cpu_file="/proc/cpuinfo"
 
             case $kernel_machine in
-                "frv" | "hppa" | "m68k" | "openrisc" | "or"* | "powerpc" | "ppc"* | "sparc"*)
+                "frv" | "hppa" | "parisc64" | "m68k" | "openrisc" | "or"* | "powerpc" | "ppc"* | "sparc"*)
                     cpu="$(awk -F':' '/^cpu\t|^CPU/ {printf $2; exit}' "$cpu_file")"
                 ;;
 
